src/separateLater/Sentinel1/Sentinel1Types.jl

```diff
@@ -0,0 +1,21 @@
+
+
+"""
+General metadata need for SAR images
+"""
+struct MetaDataSLC
```

Comment:
I noticed that readMetaData function starts with small first letter and here capitalized first letter. I am not 100% on which formatting we are using - in case it is not on purpose I just wanted to highlight :)

Reply:
I think we should go with PascalCase for modules and types (e.g. struct) and use cammelCase for the rest. That is most aligned with the core language.